rockcraft_snap_patcher/patcher.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
+
+# Rockcraft snap package patcher
+# Version 4
+
+set -xe
+
+# configurable vars
+src_snap="rockcraft_1.5.3.post51+gc99aa44.d20241011_amd64.snap" # snap package to modify
+rockcraft="." # Path to rockcraft repo
+craft_application="../craft-application/" # Path to craft_application repo
+craft_providers="../craft-providers/"  # Path to craft_providers repo
+craft_parts="../craft-parts/"  # Path to craft_providers repo
+
+
+# setup
+snap_rootfs="./.patcher/$(uuid)/"
```

Comment:
`uuid` is not installed by default on most systems.
I'd use `mktemp -d` here instead.

Reply:
Thanks :) I added the -p flag to keep the tmpdir in the local repo... It seems like the patcher breaks if the dir is in /tmp